src/blocks/kpi-chart/index.tsx

```
@@ -0,0 +1,303 @@
+}
+
+function alignEndDate(date: Date, periodType: PeriodType): Date {
+	const aligned = new Date(date.getTime());
+
+	switch (periodType) {
+		case "week":
+			// Weeks end on Sunday
+			// If day = 0 (Sun), go forward 0 days
+			// If day = 1 (Mon), go forward 6 days
+			// If day = 2 (Tue), go forward 5 days, etc.
+			const dayOfWeek = aligned.getDay();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)